### PR TITLE
fixing tests

### PR DESCRIPTION
**Pull request by crazzywizard**
### Purpose
fix ff mint staking bug

### CODE REVIEW

File: test/minter/PublicMinter.t.sol [hunk 1]
```diff
@@ -205,57 +199,7 @@ contract PublicMinterTest is DSTest, StdUtils {
         vm.stopPrank();
     }
 
-    function testRevertPublicSaleInactiveOfWalletWithTransferedMintedDiscount(
```
Comment (SweetmanTech) [inline test/minter/PublicMinter.t.sol:208]: why are we deleting this?

File: test/minter/FriendsAndFamilyMinter.t.sol
Comment (SweetmanTech) [inline test/minter/FriendsAndFamilyMinter.t.sol:1]: can you add unit test to verify tokens  minted on FriendsAndFamily minter are always `staked`?

**Review by SweetmanTech — APPROVED** [review]
2 optional points:
- unit test to verify `staked` for all `pfpMints` on `FriendsAndFamily.t.sol`
- why are we removing unit test from `PublicMinter`?

File: test/minter/PublicMinter.t.sol (hunk 1 above)
Comment (crazzywizard) [reply test/minter/PublicMinter.t.sol:208]: not valid since friends and family locks and is staked so cant transfer

**Pull request merged**